lessons/pydata/scikitlearn_api/index.ipynb

```diff
@@ -0,0 +1,1207 @@
+{
```

Comment:
<p>Tady mi také přijde overkill dělat z několika čísel DataFrame jen aby se to hezky zobrazilo v tabulce. print nebo pprint by mohli posloužit a udělat ten kód podstatně jednodušší.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>